hb_report/utillib.py

```diff
@@ -1629,6 +1630,20 @@ def time_status():
     time_f = os.path.join(constants.WORKDIR, constants.TIME_F)
     crmutils.str2file(out_string, time_f)
 
+def dump_ocfs2():
+    ocfs2_f = os.path.join(constants.WORKDIR, constants.OCFS2_F)
+    f = open(ocfs2_f, "w")
```

Comment:
Two recommendations:

1. Make sure that you check and handle the case if opening the file fails for whatever reason.

2. Use the Python `with` syntax to close the file:

```
with open(ocfs2_f, "w") as f:
   ...
```